editor/test/integration/save_test.clj

```diff
@@ -224,6 +235,9 @@
                ["/collection/all_embedded.collection" delete-first-child!]
                ["/materials/test.material" (set-prop-fn :name "new-name")]
                ["/collection/components/test.label" (set-prop-fn :text "new-text")]
+               ["/label/test.label" (set-prop-fn :text "new-text")]
+               ["/label/embedded_label.go" (set-prop-fn [0] :text "new-text")]
+               ["/label/embedded_label.collection" (set-prop-fn [0 0] :text "new-text")]
```

Comment:
This is in the `save-dirty` test. These verify that the addition of the `scale` field in `GameObject$ComponentDesc` and `GameObject$EmbeddedComponentDesc` does not cause every game object or collection file with embedded components to become dirty immediately after loading the project. It also verifies that the removal of the `scale` field from `Label$LabelDesc` do not cause the same problem.